plugins/userdata/ElectronUserData.cc

```diff
@@ -175,11 +187,13 @@ void ElectronUserData::produce(edm::Event& iEvent, const edm::EventSetup& iSetup
 			const edm::Ptr<pat::Electron> elPtr(electrons, index);
 			std::vector < uint > idCutsToInvert { 99 };
 
+			// std::cout << "Energy Correction for electron : " << index << " is: " << electronSeedCorrections(iEvent, elPtr) << std::endl;
+			el.addUserFloat("energyCorrection", electronSeedCorrections(iEvent, elPtr));
```

Comment:
I think it would make sense. However the problem is thinking how to do it. The electorns have a lot of changes applied to them:

regression->EGMSmearer->electronSeedCorrections

combining these uncertainties will be tricky, indeed im not sure if they all fully exist. Could implement a 5% unc while thinking on this. 

I beleive it is hoped that in the next reminiAOD we ewill not have to worry about regression and the electronSeedCorrections will eventually be combined into the smearer. Uncertainties will become a lot clearer at this point but its not worth holding our breathe